### ISSUE TITLE
Game Mechanic - Combat

### ISSUE THREAD

**Issue by NimNimR4r** [TeaylThorn/HighwynnGame#5]
**Expected Behaviour**
Players were expecting at least some of the enemies to have more of an individual movement instead of moving in the same way (they didn't mind too much because it was our first play test), but they would love to see more diversity
As well as: 
*  Enemies being able to attack and deal damage to the player
* Having the ability to deal damage to those enemies 

More info on current enemy movements/attacks can be seen on [Enemy Movements](https://github.com/TeaylThorn/HighwynnGame/wiki/Enemy-Movements) page 

***
**Additional Context**
Players loved the art style of the enemies, they weren't really expecting them to be as spooky/scary as they are, so they would love to see them in action!

***
**TLDR**
* Update enemy movement so that they can attack and deal damage to player 
* Update each individual enemy to be unique (each has a different movement) so that they are less stagnant/boring (e.g. capuburas swarm, boars charge, bears swipe, etc)
